application/src/main/java/com/softserve/lv460/application/controller/UserApplicationController.java

```diff
@@ -0,0 +1,49 @@
+package com.softserve.lv460.application.controller;
+
+import com.softserve.lv460.application.security.dto.JWTUserRequest;
+import com.softserve.lv460.application.security.dto.JWTUserResponse;
+import com.softserve.lv460.application.security.dto.UserRegistrationRequest;
+import com.softserve.lv460.application.security.jwt.JwtTokenProvider;
+import com.softserve.lv460.application.service.ApplicationUserService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/users")
+public class UserApplicationController {
+  private final ApplicationUserService applicationUserService;
+  private final AuthenticationManager authenticationManager;
+  private final JwtTokenProvider jwtTokenProvider;
+
+  @PostMapping("/signin")
+  public ResponseEntity<JWTUserResponse> authenticateUser(@Valid @RequestBody JWTUserRequest loginRequest) {
```

Comment:
You use annotation @Valid but u dont have any javax validation in JWTUserRequest